### ISSUE TITLE
Colored tags

### ISSUE THREAD

**Issue by Sanjinso** [hackjutsu/Lepton#484]
I really like Lepton so far.

A feature I would love to see are colored tags and tags shown in the snippet-select-panel for better overview:
![colored_tags](https://user-images.githubusercontent.com/20252587/108601260-03285780-739c-11eb-9da8-093631d1f268.jpg)

**Comment by nagalr** [comment]
I agree that this feature can be beneficial to many. 
In my opinion, an option to edit the CSS of the app (like in the Jolin notebook) could be amazing and very useful to many.

**Comment by hackjutsu** [comment]
Thanks for the suggestion and this is an interesting idea. Did you have a PR for review (given you have a screenshot already)? @Sanjinso

**Comment by Sanjinso** [comment]
@nagalr : Absolutely, loading in an custom CSS would be a great feature on its own.
@hackjutsu : I just played around with the Developer Console, couldnt get Lepton running for development purposes. Tried it with my Laragon installation and an Vagrant environment. May give it another try sometime.

**Comment by nagalr** [comment]
Great @hackjutsu @Sanjinso !
Another very useful feature that will make me leave 'Gisto' is a Search option for a keyword within the gists.
